### PR TITLE
Feat/Footer Section #issue21

### LINKED ISSUE #21: feat: footer section
### Feature Proposal

Create the Footer as the eight and last section of the landing page following the [Figma design](https://www.figma.com/community/file/1335063165996522243):

image

### PR DESCRIPTION
**Pull request by Hardsys-Andre**
Closes #21 
The Footer Section was created, following Figma Designer's instructions and responsiveness was also configured for adjustment in the Mobile version, following Figma Designer's guidance.

Below is an illustrative image of the component:
image

and version mobile:
image

### CODE REVIEW

File: src/app/page.tsx [hunk 1]
```diff
@@ -5,6 +5,7 @@ import {
 	HowItWorksSection,
 	RoadmapSection,
 } from "@/components/04-templates";
+import { TheFooterSection } from "@/components/04-templates/TheFooterSection";
```
Comment (heronlancellot) [inline src/app/page.tsx]: hey yo @Hardsys-Andre ! 
Maybe we can import TheFooterSection like the others
``@/components/04-templates"``

File: src/app/public/the-footer/instagran-icon.png
Comment (heronlancellot) [inline src/app/public/the-footer/instagran-icon.png:1]: hey yo! just change the name to Camel case
``InstagramIcon`` 
in ``public/`` for example, we are using camelCase in files

File: src/app/public/the-footer/twiter-icon.png
Comment (heronlancellot) [inline src/app/public/the-footer/twiter-icon.png:1]: same in that one 👍

File: src/app/public/the-footer/vector.svg
Comment (heronlancellot) [inline src/app/public/the-footer/vector.svg:1]: maybe we can change the name ``vector`` to another name more understandable what it is about without clicking on it 🤔 maybe ArrowUp? or another name... @Hardsys-Andre

File: src/components/01-atoms/CardTheFooterLayout.tsx
Comment (heronlancellot) [inline src/components/01-atoms/CardTheFooterLayout.tsx:1]: hey yo @Hardsys-Andre ! Beautiful component 💯

File: src/components/02-molecules/CardTheFooter.tsx
Comment (heronlancellot) [inline src/components/02-molecules/CardTheFooter.tsx:1]: hey yo ! 
- [ ] Just check on the tablet device why it is not aligned
image
&  Remove unused comment // CardsTheFooter

File: src/components/03-organisms/TheFooter.tsx
Comment (heronlancellot) [inline src/components/03-organisms/TheFooter.tsx:1]: - [ ] Remove unused code
- [ ] Adjust the imports to ``@/components/``
- [ ] update the name of the imports when updating the pngs ( ``InstagramIcon`` etc )
Also, the use of ``absolute`` in those divs doesn't look necessary.. Try to remove it and change to ``flex`` or even ``grid``
Also, check why the icon it's not aligned
image

**Review by heronlancellot — CHANGES_REQUESTED** [review]
hey yo @Hardsys-Andre ! GJ !
I made some comments * 
Update with the request changes & update the footer branch with latest changes of the ``develop`` branch! 
Nice coding ! :)

**Comment by Hardsys-Andre** [comment]
hello @heronlancellot, PR updated as per requests

**Review by heronlancellot — APPROVED** [review]
hey yo @Hardsys-Andre ! LG 🪨

**Pull request merged**